tests/repo1/github/conftest.py

```diff
@@ -0,0 +1,32 @@
+import pytest
```

Comment:
Can we please rename the file as show below?
tests/case1/github/conftest.py

Reply:
done in upcoming commit